Packs/CortexAttackSurfaceManagement/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Cortex Attack Surface Management",
     "description": "Content for working with Attack Surface Management (ASM).",
     "support": "xsoar",
-    "currentVersion": "1.5.2",
+    "currentVersion": "1.5.4",
```

Comment:
I am guessing, there is already another PR with 1.5.3?

Reply:
yes, https://github.com/demisto/content/pull/24837 is using 1.5.3 already